client/src/pages/CatalogSectionPage/CatalogSectionPage.jsx

```diff
@@ -6,28 +6,70 @@ import CategoryCardsContainer from "./components/Pagination/CategoryCardsContain
 import Pagination from "./components/Pagination/Pagination";
 
 import { useDispatch, useSelector } from "react-redux";
+import {useLocation} from "react-router-dom";
 import { useState, useEffect } from "react";
 import PropTypes from 'prop-types';
 
 import { selectorAllCollectionProduct } from "../../store/selectors";
 import { fetchAllCollectionProduct } from "../../store/actions";
 
+const addToURLWithoutReloading = (url, addConfig) => window.history.pushState(null,null, `${url}${addConfig?"/filter?"+addConfig:""}`);

```

Comment:
config is usually an object. but you use it a part of url. what do you store in `addConfig` varaible?

Reply:
also, filter is usually used as query. `/products?catrogory=rings&price=10,20`